src/Controls/src/Core/Platform/Android/Shell/ShellToolbarTracker.cs

```diff
@@ -675,11 +676,19 @@ protected override void Dispose(bool disposing)
 				}
 			}
 
-			public FlyoutIconDrawerDrawable(IMauiContext context, Color defaultColor, Drawable icon, string text) : base(context.Context)
+			public FlyoutIconDrawerDrawable(Context context, Color defaultColor, Drawable icon, string text) : base(context)
 			{
-				var fontManager = context.Services.GetRequiredService<IFontManager>();
-				TintColor = defaultColor;
-				_defaultSize = fontManager.GetFontSize(Font.OfSize("Roboto", 0));
+				TintColor = defaultColor;		
+				if (context.TryResolveAttribute(AndroidResource.Attribute.TextSize, out float? value) &&
+					value != null)
+				{
+					_defaultSize = value.Value;
+				}
+				else
+				{
+					_defaultSize = 50;
```

Comment:
Where did this 50 come from?

Reply:
Wherever this came from :-) 

https://github.com/dotnet/maui/blob/c6efff91de8fe30fadbcdfff24f7b9af96dc0c91/src/Compatibility/Core/src/Android/Forms.cs#L159

It most likely won't ever get set to 50 since the only purpose was for pre-lollipop which we don't support.

I just left here for backwards compatibility